mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -16,12 +16,11 @@
 from . import ugettext as _
 
 # logging
-from logging import getLogger
-log = getLogger(__name__)
+import logging
+log = logging.getLogger(__name__)  # type: Any
```

Comment:
Otherwise Pylance complains:
```
Cannot access member "verbose" for type "Logger"
  Member "verbose" is unknown
```

This is for sure a hack. But I dont know how to do better. Overall, I dont know why it works on master as I cant find the definition of `verbose` myself.

If you know more about it I would appreciate the help with fixing it.

Reply:
Now that I think about it - it was a bad idea to hide this error as well. Will remove the type: Any regardless of whether there is another way to fix it.

Reply:
The response I emailed was supposed to go into this thread